### PR TITLE
[CSL-1285] Add items to trackRecommendationView

### PR DESCRIPTION
**Pull request by stanlp1**
added items param to trackRecommendationView so we can use them in autocomplete-ui

### CODE REVIEW

**Review by esezen — APPROVED** [review]
LGTM!

File: spec/src/modules/tracker.js [hunk 1]
```diff
@@ -2977,6 +2977,30 @@ describe(`ConstructorIO - Tracker${bundledDescriptionSuffix}`, () => {
       expect(tracker.trackRecommendationView(clonedParameters)).to.equal(true);
     });
 
+    it('Should respond with a valid response when required parameters and items are provided', (done) => {
+      const items = [{ item_id: '123' }, { item_id: 'abc' }];
```
Comment (esezen) [inline spec/src/modules/tracker.js:2981]: Can we add a `variation_id` as well?

Comment (stanlp1) [reply spec/src/modules/tracker.js]: Done!

**Review by jjl014 — APPROVED** [review]
Lgtm! 🚀

**Pull request merged**